### PR TITLE
Feat: only allow v1 documents

### PR DESCRIPTION
**Pull request by raiseandfall**

### CODE REVIEW

File: src/constants/contexts/contexts.ts [hunk 1]
```diff
@@ -1,23 +1,11 @@
 import BLOCKCERTSV1_2 from './blockcerts.v1.2.json';
-import BLOCKCERTSV2_ALPHA from './blockcerts.v2-alpha.json';
-import BLOCKCERTSV2 from './blockcerts.v2.0.json';
-import BLOCKCERTSV3_ALPHA from './blockcerts.v3-alpha.json';
-import BLOCKCERTSV3_BETA from './blockcerts.v3-beta.json';
 
 import MERKLE_PROOF_2019 from './merkle-proof-2019.json';
```
Comment (lemoustachiste) [inline src/constants/contexts/contexts.ts:3]: not needed for v1

File: src/constants/contexts/preloadedContexts.ts [hunk 2]
```diff
@@ -6,36 +6,8 @@ const preloadedContexts = {};
...
-preloadedContexts['https://w3id.org/blockcerts/3.0-beta'] = CONTEXTS.BLOCKCERTSV3_BETA;
-
-// extra definitions
-preloadedContexts['https://w3id.org/blockcerts/schema/3.0-alpha/merkleProof2019Context.json'] = CONTEXTS.MERKLE_PROOF_2019;
-preloadedContexts['https://www.blockcerts.org/schema/3.0-alpha/merkleProof2019Context.json'] = CONTEXTS.MERKLE_PROOF_2019;
-preloadedContexts['https://w3id.org/blockcerts/3.0-alpha/merkleProof2019Context.json'] = CONTEXTS.MERKLE_PROOF_2019;
-
 /** ***** OPEN BADGES ***** **/
 preloadedContexts['https://w3id.org/openbadges/v2'] = CONTEXTS.OPEN_BADGES;
 preloadedContexts['https://openbadgespec.org/v2/context.json'] = CONTEXTS.OPEN_BADGES;
```
Comment (lemoustachiste) [inline src/constants/contexts/preloadedContexts.ts:11]: I am not even sure v1 forks off open badges?

**Review by lemoustachiste — APPROVED** [review]
I think it's good for what we want to do. I don't believe we need to set up dependabot for this project, just keep it in a state where it works.
It'd be good to remove the `dist` folder from the tracking too.

File: src/constants/contexts/preloadedContexts.ts (hunk 2 above)
Comment (raiseandfall) [reply src/constants/contexts/preloadedContexts.ts:11]: I don't even know. I want say it doesn't.

Comment (raiseandfall) [reply src/constants/contexts/preloadedContexts.ts:11]: [It does use it](https://community.blockcerts.org/t/v2-schema-highlights/32) but only the OB v1.1 standard. These contexts are for OBI v2.

**Pull request merged**